src/query/bboxquerydirective.js

```diff
@@ -1,8 +1,13 @@
-goog.provide('ngeo.bboxQueryDirective');
+/**
+ * @module ngeo query namespace
+ */
+goog.provide('ngeo.query.bboxQueryDirective');
```

Comment:
Event if that's always an angular directive, can you names it "component ?".
That's a more generic term and In the future, we will change it to a component. So we (and our customers in their custom code) will have to update again the require.

Reply:
+1